app/services/sources/sources.ts

```diff
@@ -356,7 +358,7 @@ export class SourcesService extends StatefulService<ISourcesState> {
     const source = this.views.getSource(id)!;
     const muted = obsInput.muted;
     this.UPDATE_SOURCE({ id, muted });
-    this.updateSourceFlags(source.state, obsInput.outputFlags, true);
+    this.updateSourceFlags(source, obsInput.outputFlags, true);
```

Comment:
Why remove `.state`?

Reply:
Because I need to pass `Source` instead of `ISource`